### DISCUSSION TITLE
Grape Preprunner

### DISCUSSION THREAD

**Discussion by ibeXedi** [Courseplay/Courseplay_FS22#2806] [Q&A] [answered]
Hello Coursplay community,

I found myself playing the map Silverrun Forest and decided after some forest work its time to test grapes.
After quickly noticing that I need help from courseplay ....

Everything works fine except CP misses many vines because of the offset needs to be higher in my case (1,5 - 1,6) because of the hills.
![20230925130010_1](https://github.com/Courseplay/Courseplay_FS22/assets/57322815/c2da1f58-a1bf-4916-b754-c76ab7cda35c)

As you can see on the way back it gets too far away from the vines and on the other way it wants to kiss the vines and get stuck.

I tried to manipulate the Preprunner to make the "range" higher but I failed.

Can somebody help me out with some CP settings regarding this problem or help me out fixing the Preprunner to be more generous with where vines need to be?

I used a modded preprunner since this seems to get larger when extended ^^
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=260743&title=fs2022

When I use the Fendt with the "stock" preprunner it only gets worse since the Fendt has a higher cabin and the preprunner is smaller in width.

Best Regards
ibeXedi

**Comment by Tensuko** [comment, marked as answer]
Guess why we have an adjustable offset setting on the hud.
The Vanilla should just work, when the tractor itself is too hight for the tool, then there is nothing we can do.

**Reply by ibeXedi** [reply]
Thanks for the quick response. In my opinion Farming Simulator itself is in fault here since even manualy driving is difficult.

I will try to contact the creator of the mod (Preprunner) maybe he can publish a longer version than it should be fixed.

**Comment by rerfazef5** [comment]
yes, sure.
